block-node/messaging/src/main/java/org/hiero/block/node/messaging/BlockMessagingFacilityImpl.java

```diff
@@ -374,6 +417,7 @@ public void sendBlockPersisted(PersistedNotification notification) {
      */
     @Override
     public void sendBackfilledBlockNotification(BackfilledBlockNotification notification) {
+        LOGGER.log(TRACE, "Sending backfilled block notification: " + notification);
```

Comment:
For performance, can we do the following:
```suggestion
        LOGGER.log(TRACE, "Sending backfilled block notification: {0}.", notification);
```

Reply:
✅ Done!